### PR TITLE
License level clarification

### PR DESCRIPTION
**Pull request by mhdostal**
Updates the documentation regarding Lite license level; offline editing and syncing require a Basic license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ This build rule looks for a secrets file stored in the project directory, `$(PRO
 
   > Replace 'your-license-key' with your **License Key**.
 
-A Lite license is free and enables all of the capabilities required by this app. Visit the developer's website to learn more about [licensing your ArcGIS Runtime app](https://developers.arcgis.com/pricing/licensing/).
+A Lite license is free and enables all of the capabilities required by this app, with the exception of offline editing and syncing offline edits back to a feature service.  For offline editing and syncing, you need a Basic licence. Visit the developer's website to learn more about [licensing your ArcGIS Runtime app](https://developers.arcgis.com/pricing/licensing/) and the different [ArcGIS Runtime license levels](https://developers.arcgis.com/ios/license-and-deployment/license/).
```
Comment (esreli) [inline README.md:199]:
```suggestion
A Lite license is free and enables all of the capabilities required by this app, with the exception of offline editing and synchronizing offline edits back to a feature service.  For offline editing and synchronizing, you need a Basic license. Visit the developer's website to learn more about [licensing your ArcGIS Runtime app](https://developers.arcgis.com/pricing/licensing/) and the different [ArcGIS Runtime license levels](https://developers.arcgis.com/ios/license-and-deployment/license/).
```

**Review by esreli — CHANGES_REQUESTED** [review]

**Comment by mhdostal** [comment]
@esreli - this is once again ready for your review.

File: README.md [hunk 2]
```diff
@@ -196,7 +196,7 @@ This build rule looks for a secrets file stored in the project directory, `$(PRO
 
   > Replace 'your-license-key' with your **License Key**.
 
-A Lite license is free and enables all of the capabilities required by this app. Visit the developer's website to learn more about [licensing your ArcGIS Runtime app](https://developers.arcgis.com/pricing/licensing/).
+A Lite license is free and enables all of the capabilities required by this app, with the exception of offline editing and synchronizing offline edits back to a feature service.  For offline editing and synchronizing, you need a Basic license. Visit the developer's website to learn more about [licensing your ArcGIS Runtime app](https://developers.arcgis.com/pricing/licensing/) and the different [ArcGIS Runtime license levels](https://developers.arcgis.com/ios/license-and-deployment/license/).
```
Comment (esreli) [inline README.md:199]: Two more notes here but I think we are nearly done here.

First, it feels a bit odd to me to say "a Lite license is free and enables all of the capabilities..." only to turn around and say, "with the exception of...". Maybe we can swap the word "all" for "most"?

Second, one of the links in this doc gets redirected so it's probably best that we change the URL to be that of the redirected page.

```suggestion
A Lite license is free and enables most of the capabilities required by this app, with the exception of offline editing and synchronizing offline edits back to a feature service. For offline editing and synchronizing, you need a Basic license. Visit the developer's website to learn more about [licensing your ArcGIS Runtime app](https://developers.arcgis.com/documentation/mapping-apis-and-location-services/licensing/license-your-app/) and the different [ArcGIS Runtime license levels](https://developers.arcgis.com/ios/license-and-deployment/license/).
```

**Review by esreli — APPROVED** [review]
Thanks!

**Pull request merged**